pkgs/applications/networking/aether/default.nix

```diff
@@ -0,0 +1,89 @@
+{ autoPatchelfHook, makeDesktopItem, lib, stdenv, wrapGAppsHook
+, alsa-lib, at-spi2-atk, at-spi2-core, atk, cairo, cups, dbus, expat, fontconfig
+, freetype, gdk-pixbuf, glib, gtk3, libcxx, libdrm, libnotify, libpulseaudio, libuuid
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, libxshmfence
+, mesa, nspr, nss, pango, systemd, libappindicator-gtk3, libdbusmenu
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aether";
+  version = "2.0.0-dev15";
+  binaryName = "AetherP2P";
+  desktopName = "Aether";
+
+  src = fetchurl {
+    url = "https://static.getaether.net/Releases/Aether-2.0.0-dev.15/2011262249.19338c93/linux/Aether-2.0.0-dev.15%2B2011262249.19338c93.tar.gz";
+    sha256 = "1hi8w83zal3ciyzg2m62shkbyh6hj7gwsidg3dn88mhfy68himf7";
+    name = "aether-tarball.tar.gz";
+  };
+
+  dontWrapGApps = true;
+
+  nativeBuildInputs = [
+    alsa-lib
+    autoPatchelfHook
+    cups
+    libdrm
+    libuuid
+    libXdamage
+    libX11
+    libXScrnSaver
+    libXtst
+    libxcb
+    libxshmfence
+    mesa
+    nss
+    wrapGAppsHook
+  ];
+
+  libPath = lib.makeLibraryPath [
+    libcxx systemd libpulseaudio libdrm mesa
+    stdenv.cc.cc alsa-lib atk at-spi2-atk at-spi2-core cairo cups dbus expat fontconfig freetype
+    gdk-pixbuf glib gtk3 libnotify libX11 libXcomposite libuuid
+    libXcursor libXdamage libXext libXfixes libXi libXrandr libXrender
+    libXtst nspr nss libxcb pango systemd libXScrnSaver
+    libappindicator-gtk3 libdbusmenu
+  ];
+
+  installPhase = ''
+    mkdir -p $out/{bin,opt/${binaryName},share/pixmaps}
+    mv * $out/opt/${binaryName}
+
+    chmod +x $out/opt/${binaryName}/${binaryName}
+    patchelf --set-interpreter ${stdenv.cc.bintools.dynamicLinker} \
+        $out/opt/${binaryName}/${binaryName}
+
+    wrapProgram $out/opt/${binaryName}/${binaryName} \
+        "''${gappsWrapperArgs[@]}" \
+        --prefix XDG_DATA_DIRS : "${gtk3}/share/gsettings-schemas/${gtk3.name}/" \
+        --prefix LD_LIBRARY_PATH : ${libPath}
+
+    ln -s $out/opt/${binaryName}/${binaryName} $out/bin/
+    # Without || true the install would fail on case-insensitive filesystems
+    ln -s $out/opt/${binaryName}/${binaryName} $out/bin/${lib.strings.toLower binaryName} || true
+    ln -s $out/opt/${binaryName}/aether.png $out/share/pixmaps/${pname}.png
+
+    ln -s "${desktopItem}/share/applications" $out/share/
+  '';
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = binaryName;
+    icon = pname;
+    inherit desktopName;
+    genericName = meta.description;
+    categories = "Network;";
+    mimeType = "x-scheme-handler/aether";
+  };
+
+  meta = with lib; {
+    description = "Peer-to-peer ephemeral public communities";
+    homepage = "https://getaether.net/";
+    downloadPage = "https://getaether.net/download/";
+    license = licenses.agpl3;
```

Comment:
Do you know if the license is agpl3Only or agpl3Plus?

Reply:
I do not think the Aether authors thought about that. They only copied the LICENSE file in verbatim into their repo root. I checked a couple of the source files and they did not include a version notice at all. I read through the relevant sections of the AGPL text and as far as I understand it, the raw license text does not imply only or plus.

Reply:
https://github.com/aethereans/aether-app

Reply:
From the license

> If the Program does not specify a version number of the GNU Affero General Public License, you may choose any version ever published by the Free Software Foundation.

However, the package.json refers to the file, which itself _is_ a version. This is hilarious.

We should ask them. Do we have a boilerplate for that? It would be nice to have a good explanation of the question and its relevance to this community and others.

Reply:
If anyone knows of earlier cases, they can share their experience here https://github.com/jtojnar/nixpkgs-hammering/issues/118

Reply:
Request has been made. Current value is sufficient for now.

Reply:
Hello, I'm from Aether. I have [clarified the license to be **AGPL-3.0-only**](https://github.com/aethereans/aether-app/commit/d054809743b03e0167299abb680a622888763a71) after speaking with @nehbit. Thanks!